woocommerce/woocommerce
# Update product editor tour design

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This is a follow-up to #38590 and #38640

The text, images, and styles for the product editor tour/guide have been updated to match the latest design (lj4AUMJnpMtUe7XqyvOONq-fi-1335_125302).

Closes #38705.

#### Before

<img width="1097" alt="Screenshot 2023-06-14 at 17 36 43" src="https://github.com/woocommerce/woocommerce/assets/2098816/2257f051-6714-4a2a-9965-059b667e0b09">

<img width="1097" alt="Screenshot 2023-06-14 at 17 36 49" src="https://github.com/woocommerce/woocommerce/assets/2098816/95691d36-20ac-48e2-937c-a2a9f2b79f73">


#### After

<img width="1097" alt="Screenshot 2023-06-14 at 17 34 44" src="https://github.com/woocommerce/woocommerce/assets/2098816/d43a3df6-faf3-439a-b923-07e18245ee36">

<img width="1097" alt="Screenshot 2023-06-14 at 17 34 51" src="https://github.com/woocommerce/woocommerce/assets/2098816/a229e421-a204-45db-aad3-aa5c8d67778d">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the block product editor in WooCommerce > Settings > Advanced > Features.
2. Go to Product > Add New.
3. You should see the tour kit step in the bottom left of the editor. (If you have already dismissed it, you can delete the woocommerce_block_product_tour_shown option through WCA Test Helper)
4. Check the styles against the design in lj4AUMJnpMtUe7XqyvOONq-fi-1335_125302
    * Text (titles, descriptions) -- these vary depending on whether the user is new (no published product) or not
    * Images
